### ISSUE TITLE
Ensures the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds

### ISSUE THREAD

**Issue by lizgarciao** [pulibrary/Google-Liz#4]
### Steps to recreate
Load: https://www.google.com/

### URL
https://www.google.com/

### Issues
Ensures the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds

### Severity
Serious

### WCAG Conformance level (A, AA, AAA)
WCAG 2 AA

### User impact
Users with low vision cannot differentiate the colors

### Recommended fix
To solve this problem, you need to fix the following:
Element has insufficient color contrast of 4.15 (foreground color: #70757a, background color: #f2f2f2, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1

### Acceptance criteria
- [x] I have checked this feature using one of the following browser extensions and confirmed that it does not add any level A or level AA issues:
    * axe dev tools
    * Lighthouse
    * Wave

### Notes
